src/guide/api.md

```diff
@@ -366,21 +366,53 @@ HTTP API errors will have the following response body:
 ```
 -->
 
+## `/magic-links/send`
+
+### Request
+
+`POST` requests made to the `/magic-links/send` endpoint emails the address provided with a Magic Link. This magic link contains a URL, provided by you, that will redirect the recipient to an endpoint in your application. From here, you can send the token Passwordless.dev has embedded in the link to verify the token at `signin/verify`.
+
+The request must include all three fields.
+
+- `emailAddress`: Recipient of the magic link. Must be a valid email address.
+- `urlTemplate`: This is the URL that users will be directed to when they click the link. It should be a valid URL except for the token template string, `<token>`. We will swap `<token>` with the actual token value before sending the email. In your application, you should parse the token out of the url (most easily done with a query parameter as seen below) and send it to the `signin/verify` endpoint to validate the request.
+- `userId`: The identifier of the user the email is intended for.
+
+```http request
```

Comment:
Is the `request` part required here? I didn't know language specifiers can have spaces

Reply:
Rider offered it as auto-complete. It appears to recognize it and provides different coloring. It looks the same in VuePress...so might just keep it 🤷  ?

Reply:
I think we use just plain `http` in other places so maybe makes sense to stay consistent (or update those existing instances)